nvflare/app_opt/pt/file_list_model_locator.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+from nvflare.apis.dxo import DXO
+from nvflare.apis.event_type import EventType
+from nvflare.apis.fl_context import FLContext
+from nvflare.app_common.abstract.model import model_learnable_to_dxo
+from nvflare.app_common.abstract.model_locator import ModelLocator
+from nvflare.app_opt.pt.file_model_persistor import PTFileModelPersistor
+
+
+class PTFileListModelLocator(ModelLocator):
```

Comment:
This is a generic ModelLocator that has nothing to with PT. It shouldn't even have to do with files.
In my mind, the model list is simply a dict of: name => location. It is up to the model persistor how to load from the "location".

So this class should be named without PT or File. Maybe something like ModelListLocator?

Reply:
Changed to ModelListLocator

Reply:
But the code still shows the class name to be FileListModelLocator (now that PT is removed).

Reply:
It's ListModelLocator in the app_common. can you refresh and check?